src/components/pages/Chat/ChatWindow.tsx

```diff
@@ -73,18 +74,18 @@ const FeedFlex = styled.div`
 `
 
 type FeedProps = {
-    className?: string,
-    children?: React.ReactNode,
+    className?: string
+    children?: React.ReactNode
```

Comment:
Coming from `flow` does not help haha, you use commas there. I learned today in typescript you do not. 💪🏻